### PR TITLE
06: CSS Transitions

### PR DESCRIPTION
**Pull request by gabrielizalo**
# Summay

* Updated scss file with animation.

# Trello Card

https://trello.com/c/VBUMdRHI/25-css-transitions

# Screenshot

oct-02-2018 10-35-22

### CODE REVIEW

File: maquetado/scss/_tech.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    position: absolute;
+    top: -10px;
+    transition: left 0.5s ease,
+                top 0.5s ease;
+    width: 100%;
+    z-index: -1;
+  }
+
+  &:hover:before {
+    left: 10px;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: I recommend you this [article](https://medium.com/outsystems-experts/how-to-achieve-60-fps-animations-with-css3-db7b98610108), you should avoid use the properties that trigger the `painting`, you can use the `transform` property

**Review by henryzarza — DISMISSED** [review]
Great 👍. Check my recommendation

File: maquetado/scss/_tech.scss [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  &:before {
+    border: 1px solid $green;
+    border-radius: $tech-border-radius;
+    content: '';
+    height: 100%;
+    left: -10px;
+    position: absolute;
+    top: -10px;
+    transition: left 0.5s ease,
+                top 0.5s ease;
```
Comment (mcallegari10) [inline maquetado/scss/_tech.scss]: Use `transition: all 0.5 ease;`

File: maquetado/scss/_tech.scss (hunk 1 above)
Comment (mcallegari10) [reply maquetado/scss/_tech.scss]: Check this please. The `transform` way to do the animation is the better way

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**